LondonBritishLibrary/orient/BLorient11763.xml

```diff
@@ -67,8 +67,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                             </handNote>
                         </handDesc>
                         <decoDesc>
-                            <decoNote xml:id="d1" type="miniature">
-                                <desc>One coloured magic picture.</desc>
+                            <decoNote xml:id="d1" type="drawing">
+                                <desc>One coloured <term key="Magic">magic picture</term>.</desc>
```

Comment:
In my understanding, drawing is not coloured. Or is somehow different in the Guidelines?